src/components/App.js

```diff
@@ -30,6 +31,10 @@ const App = () => {
     setInput(event.target.value)
   }
 
+  const deleteToDo = (id) => {
```

Comment:
`todo` en inglés se escribe todo en minúsculas, si bien son dos palabras separadas, se trata como si fuera una sola, por lo que por consistencia es mejor mantenerla así

```suggestion
  const deleteTodo = (id) => {
```